### DISCUSSION TITLE
Is talos.config kernel param permanent?

### DISCUSSION THREAD

**Discussion by CRASH-Tech** [siderolabs/talos#7188] [Q&A] [answered]
And is this url will be fetched periodically?
Or how set, for periodically check, like in the sidero/capi?

**Comment by smira** [comment]
By default machine config is persisted on the node, so Talos will never try to fetch the machine config as long as the `STATE` partition is not wiped.

**Reply by CRASH-Tech** [reply, marked as answer]
thank
